apps/test/util/testUtils.js

```diff
@@ -3,16 +3,10 @@ import {assert} from './configuredChai';
 require('require-globify');
 
 var $ = require('jquery');
-var React = require('react');
-var ReactDOM = require('react-dom');
-var Radium = require('radium');
 
 exports.setExternalGlobals = function () {
-  window.React = React;
-  window.ReactDOM = ReactDOM;
   window.$ = $;
   window.jQuery = $;
-  window.Radium = Radium;
```

Comment:
Here's the second kludge we get to (mostly) remove by doing this - tests don't depend on having these on window anymore.